### ISSUE TITLE
Не отображаются BTTV смайлики

### ISSUE THREAD

**Issue by Edoezinna3** [Flowseal/zapret-discord-youtube#300]
Что на сайте, что на чате Twitch выходит ошибка err_cert_authority_invalid
Пожалуйста, сделайте что-либо с этим

**Comment by e2li1386** [comment]
И FRANKERFACEZ тоже не отображаются =) В целом панель со смайлами в расширениях допустим BetterTTV, на половину срезается в плане отображения смайлов.  Соответственно на стримах тоже не отображаются, в чате.

**Comment by prake1** [comment]
Это не совсем баг проги, что забавно. Какого-то хера, в раше видимо забанили бетер тв смайлики. Когда я запустил впн на компе, то они все закешировались и дальше с запретом они уже работали. Кстати сайты 7тв в генерал листе запрета только ломают мне смайлики, когда я убрал эти строки, то они начали нормально работать.

**Comment by igrogametw** [comment]
Кто нибудь решил проблему отображения или обход какой нибудь?

**Comment by prake1** [comment]
Да, добавляешь в файл лист-генерал  все адреса основных спйтов смайлов -
есть в этой теме. Потом подбираешь тот батник из предложенных автором,
который включит и ютуб и диск и смайлики. Автор добавил много батников для
разных провайдеров.

вс, 1 дек. 2024 г., 08:08 igrogametw ***@***.***>:

**Comment by saibrok** [comment]
в конец list-general.txt нужно добавить
cdn.betterttv.net

**Comment by mamedov74** [comment]
А когда добавляешь сайт в list-general.txt нужно удалять сервис и заного устанавливать?
